tron/api/adapter.py

```diff
@@ -149,8 +154,30 @@ def _get_alternate_output_paths(self):
             if os.path.exists(formatted_alt_path):
                 yield formatted_alt_path
 
+    @toggle_flag("include_meta")
+    def get_meta(self) -> List[str]:
+        if not isinstance(self._obj, KubernetesActionRun):
+            return ["When this action is migrated to Kubernetes, this will contain Tron/task_processing output."]
+
+        filename = actioncommand.ActionCommand.STDERR
+        output: List[str] = self._get_serializer().tail(filename, self.max_lines)
+        if not output:
+            for alt_path in self._get_alternate_output_paths():
+                output = self._get_serializer(alt_path).tail(filename, self.max_lines)
+                if output:
+                    break
+        return output or []
+
     @toggle_flag("include_stdout")
-    def get_stdout(self):
+    def get_stdout(self) -> List[str]:
+        if isinstance(self._obj, KubernetesActionRun):
+            return read_log_stream_for_action_run(
+                action_run_id=self._obj.id,
+                component="stderr",
```

Comment:
Is this right? Should this be stdout not stderr?

Reply:
whoops!